common/scripts/x86_64/CentOS_7/remote/installState.sh

```diff
@@ -5,14 +5,9 @@ export SSH_TIMEOUT=60
 
 install_deps() {
```

Comment:
Completely remove the firewall installation. This is a manual step today for U14/U16. Introducing this here is probably not right and needs to be done holistically if we want to.